ksqldb-tools/src/main/java/io/confluent/ksql/tools/migrations/MigrationConfig.java

```diff
@@ -167,17 +174,20 @@ private static String getServiceId(final Map<String, String> configs) throws Mig
 
     final Client ksqlClient = MigrationsUtil.getKsqlClient(
         ksqlServerUrl,
-        configs.get(KSQL_USERNAME),
-        configs.get(KSQL_PASSWORD),
+        configs.get(KSQL_BASIC_AUTH_USERNAME),
+        configs.get(KSQL_BASIC_AUTH_PASSWORD),
         configs.get(SSL_TRUSTSTORE_LOCATION),
         configs.get(SSL_TRUSTSTORE_PASSWORD),
         configs.get(SSL_KEYSTORE_LOCATION),
         configs.get(SSL_KEYSTORE_PASSWORD),
         configs.get(SSL_KEY_ALIAS),
         configs.get(SSL_KEY_PASSWORD),
-        configs.get(SSL_ALPN).equals("true")
+        configs.getOrDefault(SSL_ALPN, "false").equals("true"),
```

Comment:
Should this check be case-insensitive? I don't remember off the top of my head how AbstractConfig handles case-sensitivity for booleans. My guess is that it's case-insensitive but we should verify.